tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: tutorial
+title: How to create and publish a new draft profile
+previousTutorial:
+  link: ./howto/howto_right_profile
+  title: How to select the right profile
+nextTutorial:
+  link: ./howto/howto_add_markup
+  title: How to add markup to your own resource
+
+bioschemas:
+  "@context": https://schema.org/
+  "@type": LearningResource
+  "http://purl.org/dc/terms/conformsTo":
+  - "@type": CreativeWork
+    "@id": "https://bioschemas.org/profiles/TrainingMaterial/0.9-DRAFT-2020_12_08/"
+  about:
+    - "@id": https://schema.org
+    - "@id": http://edamontology.org/topic_0089
+  audience:
+  - "@type": Audience
+    name: People interested in creating and publishing a new draft profile
+  name: "How to create and publish a new draft profile"
+  author:
+  - "@type": Person
+    name: "Leyla Garcia"
+    "@id": https://bioschemas.org/people/LeylaGarcia
+    url: https://bioschemas.org/people/LeylaGarcia
+  contributor:
+  - "@type": Person
+    name: "Alban Gaignard"
+    "@id": https://bioschemas.org/people/AlbanGaignard
+    url: https://bioschemas.org/people/AlbanGaignard   
+  dateModified: 2021-05-05
+  description: "In this how-to, we will guide you through the necessary steps for you to create and publish a new draft profile, you need some knowledge on spreadsheets, GitHub, git and Jekyll."
+  keywords: "schema.org, markup, structured data, bioschemas profile"
+  license: CC-BY 4.0
+  version: 1.0
+---
+
+## 1. Prepare your working environment
+
+To create and publish a new draft profile you need access to a couple of Bioschemas resources, here we list them all together with the main role they play in the whole process. We will explain how to use them and what to do with them in later steps.
+
+* Bioschemas specifications [GDrive folder](https://drive.google.com/drive/folders/0B8yXU9SkT3ftaWJtTGYyTTJjck0): to create the spreadsheet corresponding to the new draft profile, you need editing rights
+* [GOweb repository](https://github.com/BioSchemas/bioschemas-goweb): to move from the spreadsheet to the website
+* [Website repository](https://github.com/BioSchemas/bioschemas.github.io): to publish the new draft profile, you need editing rights
+* [Specifications repository](https://github.com/BioSchemas/specifications): to add compliant examples regarding the new draft profile, you need editing rights
+
+If you do not have the right permissions to edit the mentioned resources, send an email to the [mailing list](mailto:public-bioschemas@w3.org) explaining why you should have access and what Bioschemas group you belong to. Once your case has been validated, you will get an invitation to join the GDrive and the repositories.
```

Comment:
Are we expecting people to do all these steps? Are we going to offer to assist with the GoWeb process?

Reply:
Yes, we do expect people to go trough all those steps, that is how the process currently work.
Yes, it would be nice to offer some assistance on GoWeb process but, who would do so?

Reply:
OK, I'm happy that the tutorial covers all aspects.

Do we have a link for help?

Reply:
I see you have already preempted this.